src/pages/Results/Results.page.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import queryString from 'query-string';
+import { useYoutubeVideosByKeyword } from 'hooks/youtube/videos/useYoutubeVideosByKeyword';
+import VideoCard from 'components/VideoCard';
+import { ResultsWrapper } from './ResultsElements.styled';
+
+function Results() {
```

Comment:
I tested searching for video term and it worked, but the following attempts were only updating the URL path, maybe the problem is related with your dependency array on your `useEffect`